### DISCUSSION TITLE
Help Needed to Rejoin GitHub Classroom

### DISCUSSION THREAD

**Discussion by SalmonStar** [community/community#141263] [GitHub Education]
Hello everyone,

I accidentally clicked on a classmate's name while joining GitHub Classroom and ended up leaving. Now, I can't access the classroom at all. I’ve asked our TA for help, but we haven’t found a solution. I couldn't find specific instructions online for rejoining.

Could anyone please advise what steps I should take, or how our TA can assist me in getting back into the classroom?

Thank you!

**Comment by RahulScripted** [comment]
You've accidentally left the GitHub Classroom by clicking on your classmate's name. Unfortunately, once you leave, rejoining on your own isn't a direct option. However, your TA or instructor can easily help you get back into the classroom by following these steps:

1. Ask the TA or instructor to remove your name from the roster. They can access the classroom settings and remove you from the student list.
2. Re-send the invite link: After removing your name, your TA can send you a fresh invitation to rejoin the classroom. Once you accept the invitation, you should be back in.

**Reply by SalmonStar** [reply]
Thank you!

**Reply by SalmonStar** [reply]
@RahulScripted Sorry to bother you again. I would like to ask if the re-invite link you mentioned is the same link that was used to invite other classmates initially? After the TA removed and re-added me, I still encounter a 'Repository Access Issue' when using that link. I also tried another link that goes directly to my assignment repo, but it shows '404 This is not the web page you are looking for.'

**Reply by RahulScripted** [reply]
@SalmonStar  the invitation link should be a new one specifically for you after re-adding you to the repository ask your TA to generate a new link.

**Reply by RahulScripted** [reply]
@SalmonStar However, I would suggest contacting your TA again, explaining the exact error, and requesting a new invite link to ensure you get proper access.

**Reply by SalmonStar** [reply]
@RahulScripted In fact, I have already contacted the TA regarding this matter and have been in continuous communication. However, the link provided by the TA seems to be consistently invalid. I'm worried that my personal mistakes might be causing trouble for the TA, so I decided to ask for help here.
Additionally, one of the links I mentioned earlier is 'https://github.com/<course_name>/<assignment_name>-SalmonStar,' which is already assigned to me, but I still can't access it. I wanted to ask if the newly generated invitation link would follow this format?

**Reply by RahulScripted** [reply]
@SalmonStar I think it'll follow the format but not sure properly 😅 btw do you told that the new invitation link also shown invalid!??

**Reply by SalmonStar** [reply]
@RahulScripted In fact, that was a previous link, probably a direct link to the repo. However, since the TA informed me that they had removed and re-added me, they asked me to try the link again, but it is still invalid.
That's why I asked about the format of the new link. I'm really hoping that the TA simply forgot to generate a new link for me.
